src/schemas/Program/index.js

```diff
@@ -242,33 +242,19 @@ const convertGrpcUserToGql = (userDetails) => ({
 	inviteAcceptedAt: getIsoDate(get(userDetails, 'accepted_at.seconds')),
 });
 
-const formatHttpProgram = (program) => ({
-	name: program.name,
-	shortName: program.shortName,
-	description: program.description,
-	website: program.website,
-	institutions: program.programInstitutions?.map((institution) => institution.name) || [],
-	countries: program.programCountries?.map((country) => country.name) || [],
-	regions: program.processingRegions?.map((region) => region.name) || [],
-	cancerTypes: program.programCancers?.map((cancer) => cancer.name) || [],
-	primarySites: program.programPrimarySites?.map((primarySite) => primarySite.name) || [],
-});
-
-const resolveProgramList = async (egoToken) => {
-	const response = await programService.listPrograms(egoToken);
-	const programs = get(response, 'programs', []);
-	return programs.map((program) => convertGrpcProgramToGql(program));
```

Comment:
moving format handlers into the http fetch file. (maybe refactor the formatter later on)